Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -2981,14 +2981,19 @@ void CClientVehicle::Destroy()
         {
             // Only remove him physically. Don't let the ped update us
             pPed->InternalRemoveFromVehicle(m_pVehicle);
+            if (!g_pClientGame->IsGlitchEnabled(CClientGame::GLITCH_KICKOUTOFVEHICLE_ONMODELREPLACE))
+                pPed->SetWarpInToVehicleRequired(true);
         }
 
         // Remove all the passengers physically
+        bool bWarpInToVehicleRequired = !g_pClientGame->IsGlitchEnabled(CClientGame::GLITCH_KICKOUTOFVEHICLE_ONMODELREPLACE);
         for (unsigned int i = 0; i < 8; i++)
         {
-            if (m_pPassengers[i])
+            CClientPed* pPassenger = m_pPassengers[i];
+            if (pPassenger)
             {
-                m_pPassengers[i]->InternalRemoveFromVehicle(m_pVehicle);
+                pPassenger->InternalRemoveFromVehicle(m_pVehicle);
+                pPassenger->SetWarpInToVehicleRequired(bWarpInToVehicleRequired);
```

Comment:
Should this be forced false?

Reply:
No, it's not forced to false. I removed the check because I added a logical NOT  `!` 
![Capture](https://user-images.githubusercontent.com/10183157/81514953-3341c500-934b-11ea-8c47-9ffa16b4a9e6.PNG)
If IsGlitchEnabled returns true, then bWarpInToVehicleRequired is false, and if it returns false, then bWarpInToVehicleRequired is true.

Reply:
I mean, before the most recent commit, it would ONLY set true if the glitch was enabled. Now it sets it sets to true or false, depending on whether glitch was enabled

Reply:
You're right, but it still won't make any difference because when `InternalRemoveFromVehicle` is called, it will set m_bWarpInToVehicleRequired to false for the ped. 
https://github.com/multitheftauto/mtasa-blue/blob/master/Client/mods/deathmatch/logic/CClientPed.cpp#L4186

And in this case, we're calling InternalRemoveFromVehicle before SetWarpInToVehicleRequired.